ui/src/app/components/help-sidebar.tsx

```diff
@@ -628,6 +681,27 @@ export const HelpSidebar = fp.flow(
                       }}
                     >
                       <FlexColumn style={{ height: '100%' }}>
+                        {sidebarContent.showNavigation && (
+                          <FlexRow
+                            style={{
+                              padding: '1rem 0.625rem 1rem 0.625rem',
+                              borderBottom: '1px solid #979797',
+                              margin: '0rem 0.5rem',
+                              alignItems: 'center',
+                            }}
+                          >
+                            <Button
```

Comment:
Currently not an easy way of testing this functionality automatically in a unit test. The cromwell configuration panel test does not load the header and the help sidebar tests do not load the sidebar pabel.

Reply:
No longer relevant with the back button removed.